cypress/e2e/copying.cy.ts

```diff
@@ -1,4 +1,11 @@
+import { showSkipped } from "cypress/support/commands";
+
 describe('Tests about copying', () => {
+  if(Cypress.env('selection') !== "default") {
+    showSkipped()
+    return
+  }
+
```

Comment:
I thought we talked about not doing this method and instead group all e2e tests by profile in different folders.